### ISSUE TITLE
Slack Mentor feedback bot

### ISSUE THREAD

**Issue by caromelendezl** [CodeYourFuture/glasgow-class-6#33]
Volunteers Naima and Dom have developed a Slack Bot tool that allows mentors assigned Software Development Course trainees (buddy mentors, as they were last year) to respond to weekly prompts from a Slack bot and receive a follow-up visual for how their trainees are performing in relation to the rest of the class and in relation to the general course milestones. We've had real problems in the past tracking buddy mentor participation and ensuring appropriate feedback is easy to give and is given consistently - hopefully this bot mitigates some of these issues and is worth trying! Let's discuss. West Midlands are currently trialling the tool so we'll have some data in the next 3-4 weeks to review from that. 

![Image](https://user-images.githubusercontent.com/117693469/205490015-8172d88a-b4a7-4f53-b38a-fa7b07eda696.png)

Tara Quinn [6 Oct at 18:30]
Blocked as we can't trial it until we have buddy pairings to test it on!

Tandy Nicole [4 Oct at 23:40]
Is anyone interested in this? If not please lets delete the ticket

Irina Preda [11 Aug at 18:41]
anyone interested in trialling it, contact Nima or Dom

**Comment by emcglackenr1** [comment]
@caromelendezl is this the same app we were talking about earlier?

**Comment by caromelendezl** [comment]
@emcglackenr1 yes. Naima contacted us to meet up. Meeting to be booked on w/c 9th January

**Comment by emcglackenr1** [comment]
Meeting has been postponed while @caromelendezl works on the mentor/buddy set up
